docs/examples/benchmark_gpt.ts

```diff
@@ -0,0 +1,90 @@
+import type { Task } from '@epfml/discojs-core'
+import { fetchTasks, data, models } from '@epfml/discojs-core'
+import { NodeTextLoader, loadModelFromDisk } from '@epfml/discojs-node'
+import * as tf from '@tensorflow/tfjs'
+
+/**
+ * Benchmark results are reported in https://github.com/epfml/disco/pull/659
+ */
+
+async function main(): Promise<void> { 
+  // Launch a server instance
+  const url = new URL('http://localhost:8080')
+  // Fetch the wikitext task from the server
+  const tasks = await fetchTasks(url)
+  const task = tasks.get('wikitext-103')
+  if (task === undefined) { throw new Error('task not found') }  
+  
+  const BENCHMARK_TRAIN = true // if false benchmark inference
+  if (BENCHMARK_TRAIN) {
+    const config: models.GPTConfig = {
+      modelType: 'gpt-nano',
+      lr: 0.0001,
+      maxIter: 10,
+      evaluateEvery:10000,
+      maxEvalBatches: 10,
+      blockSize: 8,
+      vocabSize: 50258
+    }
+    
+    const modelType = 'gpt-nano'//['gpt-nano', 'gpt-micro', 'gpt-mini', 'gpt2']
```

Comment:
I think you exposed `GPTModelType` to type that correctly. you can use [`as const`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#const-assertions) to tell the compiler to not type the variable as `string` but as `"gpt-nano"`
```suggestion
    const modelType = 'gpt-nano' as const //['gpt-nano', 'gpt-micro', 'gpt-mini', 'gpt2']
```

Reply:
Great! thank you for the documentation, I've been searching how to circumvent this issue for a while